billing/Earthfile

```diff
@@ -0,0 +1,12 @@
+VERSION 0.6
+
+FROM alpine/git:2.36.3
+
+# Estimates monthly active users of a GIT remote repo, based on past commits
+active-users:
+  LOCALLY
```

Comment:
That was my initial goal. Still working on it. But the problem without LOCALLY is git authentication.

Reply:
No I cannot, since I need the whole commit history. In any case this is only a problem for ssh URLs to private repos.
I added some documentation that explains how to handle private repos.

Reply:
You could unshallow perhaps? `git fetch --unshallow`

Anyway, LOCALLY could be fine too, since almost everyone would already have git installed.

Reply:
Actually I removed the LOCALLY, it was only there for ssh private repos

Reply:
Oh.. but aren't private repos the most common case?